### PR TITLE
chore: add pull request template PTC-468 (#6)

### PR DESCRIPTION
**Pull request by jshattu**
- add PR template to master

https://genesisglobal.atlassian.net/browse/PTC-468

-tested that the pr template doesn't end up in the genertaed seed

### CODE REVIEW

File: .github/pull_request_templae.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+[Review Guidance](https://genesisglobal.atlassian.net/wiki/spaces/ON/pages/2682257431/Platform+Code+Review+Process)
```
Comment (skawian) [inline .github/pull_request_templae.md:1]: is this link accessible to anyone?

Comment (jshattu) [reply .github/pull_request_templae.md:1]: good spot, no it's not, wasn't migrated over to notion

**Comment by jshattu** [comment]
@skawian can you access the new link,

File: .github/pull_request_templae.md (hunk 1 above)
Comment (jshattu) [reply .github/pull_request_templae.md]: it was actually migrated to notion, hard to find though

**Review by skawian — APPROVED** [review]

File: .github/pull_request_templae.md (hunk 1 above)
Comment (skawian) [reply .github/pull_request_templae.md]: worth considering if we want to make it public in case someone outside our org wanted to contribute

**Pull request merged**